test/blackbox-tests/test-cases/directory-targets/installed-dependency.t

```diff
@@ -25,7 +25,7 @@ Allow directories to be installable
   Leaving directory 'a'
 
   $ cat a/_build/install/default/lib/foo/dune-package
-  (lang dune 3.18)
+  (lang dune 3.19)
```

Comment:
This change seems unrelated. It doesn't seem to use any new features, thus it can continue to use dune-lang 3.18.

Reply:
yea, this had to be done due to version bump. I guess these tests were updated now to use XXX for versioning. Resolving